dotnet/src/Connectors/Connectors.Amazon/Bedrock/Core/Models/AI21Labs/AI21JurassicPenalties.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.Amazon.Core;
+
+/// <summary>
+/// Penalty struct for AI21 Jurassic. Accessed by both the execution settings and request object so must be kept public.
+/// </summary>
+[Serializable]
+
+public class AI21JurassicPenalties
```

Comment:
All within `Core` should be internal

```suggestion
internal class AI21JurassicPenalties
```

Reply:
This class is used for prompt execution settings. Will move outside of core!